src/main/java/org/dsmhack/repository/ReportRepository.java

```diff
@@ -0,0 +1,28 @@
+package org.dsmhack.repository;
+
+import org.dsmhack.model.ReportData;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.jdbc.core.BeanPropertyRowMapper;
+import org.springframework.jdbc.core.JdbcTemplate;
+import org.springframework.stereotype.Repository;
+
+import java.util.List;
+
+
+@Repository
+public class ReportRepository {
+
+    @Autowired
+    private JdbcTemplate jdbcTemplate;
+
+    public List<ReportData> findAllReportingInformation(String organizationId) {
```

Comment:
I'm wondering if I can do this with some jpa magic. I tried a way that @ajdinhedzic suggested with `@Query` (I'd have to dig through history to find the example) but ran into some issue; I can't remember what it was.